### ISSUE TITLE
Organization contact e-mail

### ISSUE THREAD

**Issue by pathmapper** [maputnik/editor#700]
On https://github.com/maputnik there is still the contact from @lukasmartinelli. Is this still a way to contact us? If not we should change or remove that e-mail address:

![grafik](https://user-images.githubusercontent.com/20856381/83605296-f3e84c00-a577-11ea-8be6-fb80567d131f.png)

**Comment by orangemug** [comment]
Yeah I agree, on a slightly related note we should probably add a 'code of conduct' to the repo/org also, which would need email address(es).

What would be really cool is if there was a service that would act as an 'email group' for GitHub. So for example if someone emailed `maputnik@example.com` that would then forward those emails to the core maintainers. Does anybody know if something like that exists?

**Comment by pathmapper** [comment]
> Does anybody know if something like that exists?

What about setting up an email address using an email provider and forward incoming emails to maintainers?
This would be possible e.g. with https://posteo.de/en, costs 1€/month. 
`maputnik@posteo.net` is currently available: https://posteo.de/en/signup
